aws/aws-how-to/kubernetes/deploy-ubuntu-cluster-with-eks-ami.rst

```diff
@@ -0,0 +1,65 @@
+Deploy an Ubuntu EKS cluster
+============================
+
+This guide shows how to deploy an Ubuntu EKS cluster using an EKS AMI.
+
+Prerequisites
+-------------
+
+You need:
+
+- ``eksctl`` (version 0.190 or newer): Check the instructions to `install eksctl`_
+- ``kubectl``: Check the instructions to `install kubectl`_
+
+
+Create the ``eksctl`` config file
+---------------------------------
+
+Create a ``config.yaml`` with the following content:
+
+..  code-block:: yaml
+
+    apiVersion: eksctl.io/v1alpha5
+    kind: ClusterConfig
+    metadata:
+        name: your-cluster
+        region: us-east-1
+    nodeGroups:
+    - name: ng-cluster
+      instanceType: m5.large
+      amiFamily: Ubuntu2204
+      ami: ami-ubuntu
+      overrideBootstrapCommand: |
+        #!/bin/bash
+        /etc/eks/bootstrap.sh <cluster-name>
+
+To use Ubuntu specific AMI, set ``amiFamily`` field with `Ubuntu2004` (for EKS <= 1.29) or `Ubuntu2204` (for EKS >= 1.29). For Ubuntu nodes, ``overrideBootstrapCommand`` is required for both managed and self-managed group.
+
+For further cluster customisation, check out `eksctl details`_ 
```

Comment:
i think linking to the config schema is more appropriate here. can we update the link to use `https://eksctl.io/usage/schema/`?

Reply:
Link is replaced with schema page. Thanks!